### ISSUE TITLE
`formAction` successPath not respecting browser history

### ISSUE THREAD

**Issue by brunocrosier** [SeasonedSoftware/remix-forms#26]
If you visit the [Custom Response example,](https://remix-forms.seasoned.cc/examples/actions/custom-response) then click on the [Redirect example](https://remix-forms.seasoned.cc/examples/actions/redirect), and submit the form, you will be redirected to the `/success` page

However, if you then click the "Back" button in your browser, you are taken to the Custom Response example (instead of the Redirect example)

**Comment by NhielSalvana** [comment]
I think `redirect` _replaces_ the history stack by design.

**Comment by NhielSalvana** [comment]
@brunocrosier  add `replace` in `Form` props

**Comment by NhielSalvana** [comment]
@brunocrosier I'm sure this is not best practice but in my case, I patched this using `patch-package`.

`npm install -D patch-package postinstall-postinstall`

go to root directory
make a folder named `patch` inside it have a file named `remix-forms+0.16.1.patch`

inside `/patch/remix-forms+0.16.1.patch`

is

`diff --git a/node_modules/remix-forms/lib/Form.js b/node_modules/remix-forms/lib/Form.js
index 616a327..4a4f3ea 100644
--- a/node_modules/remix-forms/lib/Form.js
+++ b/node_modules/remix-forms/lib/Form.js
@@ -69,7 +69,7 @@ function Form({ component = react_2.Form, fetcher, mode = 'onSubmit', renderFiel
         onTransition && onTransition({ transition, ...form });
     }, [transition.state]);
     const onSubmit = (event) => {
-        form.handleSubmit(() => submit(event.target, { replace: true }))(event);
+        form.handleSubmit(() => submit(event.target, { replace: false }))(event);
     };
     const Field = (0, react_1.useMemo)(() => (0, createField_1.default)({
         register: form.register,
`

add this to `scripts` in `package.json`

`"scripts:{
   ....
  "postinstall": "patch-package"
  ....
  }
  `
  
  rm -rf `node_modules`

then lastly npm install

**Comment by brunocrosier** [comment]
is the code for the docs website (https://remix-forms.seasoned.cc) open source? if so, can you send a link to the repo?

**Comment by NhielSalvana** [comment]
@brunocrosier  here [](https://github.com/SeasonedSoftware/remix-forms-site)

@sobrinho If you're referring to me. I still don't know why they opted for the default behavior of  having `useSubmit`  `replace` the browser stack, they might have future plans and is by design.  So I'll wait for the author to be back before opening a PR :)

**Comment by sobrinho** [comment]
@NhielSalvana I'm here in behalf of Seasoned, open the PR and I open an internal discussion to define if we should merge it! ;)

**Comment by danielweinmann** [comment]
Hey, guys! Sorry for the silence: I had to focus on other things for a few days.

@brunocrosier, here goes the [code for the site](https://github.com/SeasonedSoftware/remix-forms-site). We plan to move it to this repo soon, so we can have a monorepo with the package + the site. We've been using the site as our dev environment to add E2E tests. In the meantime, feel free to use it on the other repo.

@NhielSalvana, the `{ replace: true }` on the submit was there for the first use-case I had for Remix Forms but it doesn't make any sense anymore. I fixed it and published it on `0.16.2`.
